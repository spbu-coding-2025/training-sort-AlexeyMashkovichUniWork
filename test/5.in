-698155878 931780090 -698409420 -19401134 936952530 -952797039 -185515407 894025453 230162954 -576344216 801188284 -27298927 117923502 1063074167 10119043 -582701825 868056305 -688231357 701643190 491334006 -855001907 516282640 -320656643 -34225465 362962467 327683499 159747379 735655539 898173963 -244226247 -661031377 -873723740 -386187981 -285698974 180616949 -522977276 -164754189
