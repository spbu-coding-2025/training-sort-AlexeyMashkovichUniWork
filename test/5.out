-952797039 -873723740 -855001907 -698409420 -698155878 -688231357 -661031377 -582701825 -576344216 -522977276 -386187981 -320656643 -285698974 -244226247 -185515407 -164754189 -34225465 -27298927 -19401134 10119043 117923502 159747379 180616949 230162954 327683499 362962467 491334006 516282640 701643190 735655539 801188284 868056305 894025453 898173963 931780090 936952530 1063074167
